src/clj/clojurians_log/routes.clj

```diff
@@ -26,6 +26,45 @@
       (println (.isEqual fetch-date page-date))
       (.isEqual fetch-date page-date))))
 
+(defn log-route
+  [endpoint request]
+
+  (let [config @(get-in endpoint [:config :value])
+        conn (get-in endpoint [:datomic :conn])
+        {:keys [channel date ts-str]} (:route-params request)
+        cache-time (get-in config [:message-page :cache-time] 0)]
+
+
+    ;; Since we're displaying a log, presumably, all of the content is permanently cachable
+    ;; Message page content also most likely have not changed and does not require any processing/page-generation.
+    ;; In those cases, skip page generate entirely, instead of generating the contents, then have the ring
+    ;; middleware respond with a 304.
+    (if (and (not (zero? cache-time))
+             (not (message-page-might-have-updated? date (get-in request [:headers "if-modified-since"]))))
+      (-> (ring.util.response/response nil)
+          (ring.util.response/status 304))
+
+      (let [db (d/db conn)
+            messages (queries/channel-day-messages db channel date)
+            target-msg-ts (when ts-str (time-util/inst-id->inst ts-str))
```

Comment:
Just to clarify, are you saying you'd rather end up with urls that look like:
http://hostname/aleph/2018-02-07/1518007552.000076

instead of the current
http://hostname/aleph/2018-02-07/inst-2018-02-07T15:45:52.000076Z

Reply:
Yes!

Reply:
Url format changed in new commit.